src/control/security/grpc_certs.go

```diff
@@ -44,9 +44,16 @@ func GetServerTransportCredentials(cfg *TransportConfig) (credentials.TransportC
 	}
 
 	tlsConfig := tls.Config{
-		ClientAuth:   tls.RequireAndVerifyClientCert,
-		Certificates: []tls.Certificate{*cfg.tlsKeypair},
-		ClientCAs:    cfg.caPool,
+		ClientAuth:               tls.RequireAndVerifyClientCert,
+		Certificates:             []tls.Certificate{*cfg.tlsKeypair},
+		ClientCAs:                cfg.caPool,
+		MinVersion:               tls.VersionTLS12,
+		MaxVersion:               tls.VersionTLS12,
```

Comment:
Why MaxVersion? Seems like if TLS 1.3 can be negotiated it should be allowed, right?

Reply:
These are TLS 1.2 cipher suites.